godtools/GTConstants.swift

```diff
@@ -8,7 +8,6 @@
 
 import Foundation
 
-class GodToolsConstants {
-    
-    static let kApiBase = "https://mobile-content-api-stage.cru.org"
+struct GTConstants {
+    let kApiBase = "https://mobile-content-api-stage.cru.org"
```

Comment:
So it would be `static let kApiBase = "https://mobile-content-api-stage.cru.org"`